src/utils/VideoRelevanceManager.ts

```diff
@@ -25,29 +25,45 @@ export class VideoRelevanceManager {
 
   async updateVideoRelevanceValue(em: EntityManager, forceUpdateAll?: boolean) {
     if (this.videosToUpdate.size || forceUpdateAll) {
-      const [newnessWeight, viewsWeight, commentsWeight, reactionsWeight] = await config.get(
-        ConfigVariable.RelevanceWeights,
-        em
-      )
+      const [
+        newnessWeight,
+        viewsWeight,
+        commentsWeight,
+        reactionsWeight,
+        [joystreamTimestampWeight, ytTimestampWeight],
```

Comment:
To preserve backward-compatibility with config exported from the previous version of Orion:
```suggestion
        [joystreamTimestampWeight, ytTimestampWeight] = [7, 3],
```